dropwizard-index-page/build.gradle

```diff
@@ -9,9 +9,14 @@ dependencies {
     testCompile "io.dropwizard:dropwizard-assets:$dropwizardVersion"
     testCompile "io.dropwizard:dropwizard-testing:$dropwizardVersion"
     testCompile "junit:junit:$junitVersion"
-    testCompile "org.mockito:mockito-all:$mockitoVersion"
+    testCompile "org.hamcrest:hamcrest-library:$hamcrestVersion"
```

Comment:
Encountered a missing-method error due to classpath pollution by mockito-all, so I replaced it with unbundled versions.